api/timeline/v1/timeseries.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package clutch.timeline.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/timeline/v1;timelinev1";
+
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+
+message TimeseriesData {
+  string project_name = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // For a range, both start time and end time will be present
+  // For a singular event, only start time will be filled in.
+  int64 start_time_millis = 2 [ (validate.rules).int64.gt = 0 ];
+  int64 end_time_millis = 3;
```

Comment:
What if the event is just a moment in time and not a range, how would handle that situation?

Reply:
My plan for that was end_time_millis would be 0

Reply:
They could also be equal. e.g. a moment is just the start and end at the same time